### ISSUE TITLE
NZBGet won't start

### ISSUE THREAD

**Issue by PP1979NL** [SynoCommunity/spksrc#1824]
This mornig I noticed my MZBGet stopped working.
I tried to execute the package again, but immediately after that the package stops again.
When I show the log, it says nothing about any problem.

I have not updated anything or so.

I searched on this forum and found a threat with almost the same problem (https://github.com/SynoCommunity/spksrc/issues/1608)
But the there mentioned options don't give the wanted result.
Below is the text copy pasted from the Putty SSH connection

login as: admin
admin@192.168.2.5's password:
Could not chdir to home directory /var/services/homes/admin: No such file or directory

BusyBox v1.16.1 (2015-06-29 18:12:44 CST) built-in shell (ash)
Enter 'help' for a list of built-in commands.

DiskStation> /var/packages/nzbget/scripts/start-stop-status start
Starting NZBGet ...
su: must be suid to work properly
DiskStation> /var/log/messages
-sh: /var/log/messages: Permission denied
DiskStation> /var/log/messages
-sh: /var/log/messages: Permission denied
DiskStation> /var/packages/nzbget/scripts/start-stop-status status
NZBGet is not running
DiskStation> /usr/local/nzbget/var/nzbget.pid
-sh: /usr/local/nzbget/var/nzbget.pid: not found
DiskStation> /usr/local/nzbget/var/nzbget.log
-sh: /usr/local/nzbget/var/nzbget.log: Permission denied
DiskStation>

Pls could someone help me out?

**Comment by Dr-Bean** [comment]
`su: must be suid to work properly` is because you should log in as `root`, not `admin`. Both accounts share the same password.

Then, the command `/var/log/messages` won't work: you'll see an error regardless of the account you're using.
Instead, use `vi usr/local/nzbget/var/nzbget.log`, or `tail usr/local/nzbget/var/nzbget.log` (that only shows latest entries). Alternatively, you could consider a graphical interface, such as WinSCP.

Lastly, if the logs don't show anything useful enter this: `su - nzbget -c "PATH=/usr/local/nzbget/bin:${PATH} /usr/local/nzbget/bin/nzbget -c /usr/local/nzbget/var/nzbget.conf -D"` and show the result.

**Comment by PP1979NL** [comment]
Hi Dr-Bean,

Thx for the reply. Sorry for being such a big noob on this.
I will try WinSCP later on.. 

This is what I get when I try the things above. I'm doing something wrong but have no idea what.
thx in advance for helping me out.

login as: root
root@192.168.2.5's password:


BusyBox v1.16.1 (2015-06-29 18:12:44 CST) built-in shell (ash)
Enter 'help' for a list of built-in commands.

DiskStation> /var/packages/nzbget/scripts/start-stop-status start
Starting NZBGet ...
DiskStation> /var/log/messages
-ash: /var/log/messages: Permission denied
DiskStation> /var/packages/nzbget/scripts/start-stop-status status
NZBGet is not running
DiskStation> /usr/local/nzbget/var/nzbget.pid
-ash: /usr/local/nzbget/var/nzbget.pid: not found
DiskStation> tail /usr/local/nzbget/var/nzbget.log
DiskStation> su - nzbget -c
su: option requires an argument -- 'c'
BusyBox v1.16.1 (2015-06-29 18:12:44 CST) multi-call binary.

Usage: su [OPTIONS] [-] [USERNAME]

Change user id or become root

Options:
        -p,-m   Preserve environment
        -c CMD  Command to pass to 'sh -c'
        -s SH   Shell to use instead of default shell

DiskStation> "PATH=/usr/local/nzbget/bin:${PATH} /usr/local/nzbget/bin/nzbget -c
 /usr/local/nzbget/var/nzbget.conf -D"
-ash: PATH=/usr/local/nzbget/bin:/sbin:/bin:/usr/sbin:/usr/bin:/usr/syno/sbin:/usr/syno/bin:/usr/local/sbin:/usr/local/bin /usr/local/nzbget/bin/nzbget -c /usr/local/nzbget/var/nzbget.conf -D: not found
DiskStation>

**Comment by Dr-Bean** [comment]
The last command should all be on one line. Copy/paste this, and make sure it's executed as one command:
```
su - nzbget -c "PATH=/usr/local/nzbget/bin:${PATH} /usr/local/nzbget/bin/nzbget -c /usr/local/nzbget/var/nzbget.conf -D"
```

As a suggestion, it might be easier for you to uninstall and reinstall nzbget. That does mean you'll need to reconfigure it from scratch.

**Comment by PP1979NL** [comment]
Btw on your suggestion I used WinSCP and opened the logfile. It was 0Kb. There was nothing in it at all.
Is it possible to make a backup of the configuration before reinstalling NZBGet?

**Comment by Dr-Bean** [comment]
NZBget configuration is located at `/usr/local/nzbget/var/nzbget.conf`. You can make a backup, but I would suggest to not immediately use it: if your problem is caused by your configuration, then you're back where you started.

So my suggestion is to backup that file if you want, reinstall NZBget, and start with the default configuration where you can. Make sure permissions are correct (allow `users` group read/write access on the download directories), and see if NZBget is running correctly. You should be able to work it out from there.

**Comment by PP1979NL** [comment]
Again thx for your input.
I have created a group called "Media-accounts" I have added the users of NZBGet, Sonarr and CP to that group and gave it all the needed rights. If I reinstall Sonarr will the user be the same or is there a new userid created with the same name?
I have set those rights a very long time ago. Could you tell me where to find the file containing that user information?

**Comment by Dr-Bean** [comment]
I'm sorry, but you're on your own with that one, we don't provide support when our package users have been altered.

**Comment by PP1979NL** [comment]
No I have not changed your user. I have put your user in a group with all needed rights to create folders and so on. I did do the reinstallation and it is working fine now again. I tried to put the config file back and the installations seems to hold up. :-)
I only need to set the userrights correct so it can create folders again, that is why I asked if you knew where the file for that is being kept.
